sentinel-adapter/sentinel-zuul-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/zuul/fallback/BlockResponse.java

```diff
@@ -61,12 +69,14 @@ public void setRoute(String route) {
         this.route = route;
     }
 
+    @Override
+    public BlockResponse put(String key, Object value) {
+        super.put(key, value);
+        return this;
+    }
+
     @Override
     public String toString() {
-        return "{" +
-                "\"code\":" + code +
-                ", \"message\":" + "\"" + message + "\"" +
-                ", \"route\":" + "\"" + route + "\"" +
-                '}';
+      return  JSON.toJSONString(this);
```

Comment:
It's not recommended to introduce JSON libraries directly here.

Reply:
Okay, I modify it

Reply:
> It's not recommended to introduce JSON libraries directly here.

If you don't use JSON, how to parse the user-defined complex response content? Do you have a better way to deal with it?